tests/system/src/test/java/test/robot/javafx/scene/DoubleShortcutProcessing.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package test.robot.javafx.scene;
+
+import com.sun.javafx.PlatformUtil;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.input.KeyEvent;
+import javafx.scene.input.KeyCode;
+import javafx.scene.layout.VBox;
+import javafx.scene.robot.Robot;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
+
+// When a key equivalent closes a window it can be passed
+// to the new key window and processed twice.
+public class DoubleShortcutProcessing {
```

Comment:
When I launch the test with: 
```
sh gradlew -PUSE_ROBOT=true -PFULL_TEST=true :systemTests:test --tests=test.robot.javafx.scene.DoubleShortcutProcessing
```
I get the system dialog "Accessibility Access (Events)" with the message "'java' would like to control this computer using accessibility features."

After adding java to Security&Privacy->accessibility, the test works fine.

Maybe this need to be added to the javadoc of the test.

Reply:
I wouldn't know what to write. On my system adding 'java' doesn't fix the problem, the OS insists I have to add 'Terminal'. In general it's easier to bump into this dialog and then go to System Preferences since the OS will have added the executable to the list which is convenient for a file like /usr/bin/java that's normally hidden.

Reply:
This requirement is independent of this fix and applies to all robot tests. I do not think any additional documentation is needed here.